.github/workflows/android.yml

```diff
@@ -3,13 +3,13 @@ name: Android Build
 # Controls when the workflow will run
 on:
   # Triggers the workflow on push or pull request events but only for the main branch
-  push:
-     branches: 
-     - vaccinationTabIconChange
+  # push:
+  #   branches: 
+  #   - main
     
-  # pull_request:
-  #   branches:
-  #     - main
+  pull_request:
+    branches:
+      - main
```

Comment:
You do not need these line for main anymore.

Reply:
bhumika will update yml file based on the latest code